projects/baby_driver/scripts/can_util.py

```diff
@@ -183,3 +183,35 @@ def next_message(
         ))
 
     return msg
+
+def can_pack(data_list):
+    """
+    Converts list of tuples in form ((int) val, (int) len_in_bytes)
+            and combines them into a bytearray rendition
+    Returns:
+        A bytearray object representing message components passed in
+    Raises:
+        ValueError: if tuple has more than 2 items
```

Comment:
Generally it's not recommended to document the behaviour of a function under conditions breaking its public API, because then paradoxically that becomes part of the public API. This is a rather philosophical point and doesn't matter much for this case, but you might consider removing this.